src/platforms/web/server.js

```diff
@@ -1,12 +1,74 @@
+const { readFileSync, writeFileSync } = require('fs');
 const express = require('express');
+const fetch = require('node-fetch');
 const path = require('path');
 const app = express();
 const port = 1337;
+const headRegex = /(<head>).*(<\/head>)/g;
 
 app.use(express.static(__dirname));
 
-app.get('*', function(req, res) {
-  res.sendFile(path.join(__dirname, '/index.html'));
+app.get('*', async(req, res) => {
+  let finalHtml = '';
+  let identifier = req.url.split('/').pop();
+  let isAppPage = req.url.includes('/$/');
+  let isChannel = req.url.includes('/@');
+  let queryUrl = '';
+
+  switch (true) {
+    case isAppPage:
+      res.sendFile(path.join(__dirname, '/index.html'));
+      break;
+
+    case isChannel:
+      queryUrl = `https://chainquery.lbry.io/api/sql?query=SELECT channel.name as channel, claim.name, claim.description, claim.language, claim.thumbnail_url, claim.title FROM claim left join claim channel on claim.publisher_id = channel.claim_id where claim.claim_id="${identifier}"`;
```

Comment:
@sayplastic (or @nikooo777?) is there a better way for @NetOperatorWibby to be getting data from chainquery for lbry.tv than this?

Reply:
if this is serverside we can use a mysql connector, otherwise no, unless we proxy the calls.
Also this risks getting throttled by chainquery, I think there is a limit to how many queries can be sent to the web API

Reply:
How do I improve this?

Reply:
as first version we agreed that this is fine. however please update from .io to .com